contact-us.html

```diff
@@ -2,78 +2,194 @@
+            </div>
+            <label class="form-checkbox">
+                <input class="input-hidden form-checkbox__input" type="checkbox" required>
+                <span class="form-checkbox__inner">
+                    <span class="form-checkbox__box"></span>
+                    <span class="form-checkbox__text">By clicking on the 'Send Message' button I agree to <a href="" target="_blank">all the terms aof the security policy</a></span>
+                </span>
+            </label>
+            <button class="form-control__submit" type="submit">Send Message</button>
+        </form>
+    </section>
+    <footer class="footer">
+        <div class="main-footer">
+          <div class="main-footer__left">
+            <img src="img/logo_blue.svg" alt="Binabox" class="main-footer__logo" width="188" height="42">
+            <p class="main-footer__text">We are the best way to check the rarity of NFT collection.</p>
+          </div>
+          <div class="main-footer__right">
+            <div class="main-footer__right__subscribe">
+              <h3 class="main-footer__right__header">Subscribe</h3>
+              <ul class="main-footer_nav-list">
+                <li class="nav-items"><a href="" class="home-link"></a>Home</li>
+                <li class="nav-items"><a href="blog-article.html" class="blog-link"></a>Blog</li> 
+                <li class="nav-items"><a href="" class="catalog-link"></a>Catalog</li>
+                <li class="nav-items"><a href="contact-us.html" class="contact-link"></a>Contact Us</li>
+              </ul>
+            </div>
+            <div class="main-footer__right__company">
+              <h3 class="main-footer__right__header">Company</h3>
+              <ul class="main-footer_nav-list">
+                <li class="nav-items"><a href="" class="tnc-link"></a>Terms and Conditions</li>
+                <li class="nav-items"><a href="" class="privacy-link"></a>Privacy Policy</li>
+                <li class="nav-items"><a href="" class="whitepaper-link"></a>Whitepaper</li>
+                <li class="nav-items"><a href="" class="help-center-link"></a>Help Center</li>
+              </ul>
+            </div>
+          </div>
         </div>
-        <div>
-            <label for="checkbox-id">By clicking on the 'Send message' button I agree to <span> all the terms of the security policy</span></label>
-                <input type="checkbox" name="terms" id="checkbox-id">
+        <div class="bot-footer">
+          <div class="bot-footer__left">
+            <p class="bot-footer__text">BINABOX 2022 - ALL rights reserved</p>
+          </div>
+          <div class="bot-footer__right">
+            <ul class="social-media__list">
+              <li class="social-media__icon">
+                <a href="https://x.com" class="social-media-link">
+                  <img src="img/twitter-icon.svg" alt="Twitter" class="twitter-link" width="40" height="40">
```

Comment:
@Alinchik1 you are absolutely right - we use "svg" format for icons, but in case of usage icons inside "a" tag we most probably need a possibility to color this icon.
And we can change icon color only when we insert it as SVG tag.
And this way has one more advantage - if you put it as a SVG tag, then the browser don't need to download the image from the server.

Reply:
So please replace IMG tag with SVG tag for all icons.

Reply:
You can also use SPRITE to join all your icons.

Reply:
I will change the img to svg and lets speak about SPRITE next session please

Reply:
I will need your help here